wfl/utils/save_calc_results.py

```diff
@@ -82,6 +83,22 @@ def save_calc_results(atoms, *, prefix, properties):
     if 'energies' in properties:
         atoms_results['energies'] = atoms.get_potential_energies()
 
+
+    # Additional properties asked for in calculation and present in 
+    # .results, but without a specific getter function  
+    remaining_properties = [prop for prop in properties 
+                            if prop not in per_atom_properties 
+                            and prop not in per_config_properties]
+
+    for prop_name in remaining_properties:
+        prop = atoms.calc.get_property(prop_name, allow_calculation=False)
+
+        per_atom = is_per_atom(prop, num_atoms=len(atoms))
```

Comment:
Trying to match the shape and guess whether a property is per-atom is a bit flimsy, but only one of the he `implemented_properties` would have made past `calc.get_property`.